.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
+*                                   @MetaMask/shared-libraries-engineers
```

Comment:
Following standup this morning, maybe we want to add:

``` suggestion
*                                   @MetaMask/shared-libraries-engineers @Mrtenz 
```

or even:

``` suggestion
*                                   @MetaMask/shared-libraries-engineers @MetaMask/snaps-devs 
```

?

Reply:
I think adding Snaps Devs is fine, and we can add other teams if they want to?